modules/navier_stokes/doc/content/syntax/Modules/NavierStokesFV/index.md

```diff
@@ -0,0 +1,18 @@
+# FiniteVolumeNavierStokes System
+
+## Overview
+
+!! Replace this line with information regarding the FiniteVolumeNavierStokes system.
+
+!alert note
+For the finite volume implementation of the incompressible Navier Stokes equations, see [insfv](navier_stokes/insfv.md).
+
+## Example Input File Syntax
+
+!! Describe and include an example of how to use the IncompressibleNavierStokes system.
```

Comment:
add more

Reply:
I added 3 examples with detailed explanations: 
1 incompressible lid-driven cavity, 
2 porous medium incompressible flow in a channel, with ambient convection and a solid phase
3 weakly compressible fluid flow with a temperature equation